dashboard/app/controllers/followers_controller.rb

```diff
@@ -25,14 +25,25 @@ def student_register
       return render 'student_user_new', formats: [:html]
     end
 
-    Retryable.retryable on: [Mysql2::Error, ActiveRecord::RecordNotUnique], matching: /Duplicate entry/ do
-      if @user.save && @section&.add_student(@user)
-        sign_in(:user, @user)
-        redirect_to root_path, notice: I18n.t('follower.registered', section_name: @section.name)
-        return
+    if current_user && current_user.display_captcha? && !verify_recaptcha
+      current_user.increment_section_attempts
+      flash[:alert] = I18n.t('follower.captcha_required')
+      # Concatenate section code so user does not have to type section code again
+      # Note that @section will always be defined due to validations in load_section
+      inputted_section_code = @section ? @section.code : ''
+      redirection = request.path + '/' + inputted_section_code
```

Comment:
Yep definitely -- I think I included this more a safeguard in case my Rails knowledge failed me, but I believe the declaration of `@section` in `load_section` creates an instance variable that is accessible globally.